packages/swing-store/src/bundleStore.js

```diff
@@ -77,18 +77,22 @@ export function makeBundleStore(db, ensureTxn, noteExport = () => {}) {
     ensureTxn();
     const { moduleFormat } = bundle;
     if (bundleID.startsWith('b0-')) {
-      if (moduleFormat !== 'nestedEvaluate')
-        throw Fail`unsupported b0- module format ${q(moduleFormat)}`;
+      if (moduleFormat !== 'nestedEvaluate') {
+        throw Error(`unsupported b0- module format ${q(moduleFormat)}`);
```

Comment:
I think I misread your comment above and didn't realize you literally intended to use `throw Error(...)`.  I thought the question was how to best cope with needing the `if`, not how the exception should be expressed.  The ``throw Fail`...` `` patten has gotten familiar pretty quickly, and it seems like we're backsliding on our convention here.